Templates/TodoList-NodeJS/function.json

```diff
@@ -0,0 +1,33 @@
+﻿{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"
+    },
+    {
+      "type": "table",
```

Comment:
Ideally this function would be just an empty mapper function. That calls other functions specific to each http method. This template is just a way to provide possible things to do with templates. We would have a corresponding sample in the samples repo that shows how to do this using several functions. I can may be add some comments in the code and readme.md saying that a better way to do this would be with seperate functions and point to alink in the samples repo?